pkg/infoschema/infoschema_v2.go

```diff
@@ -49,6 +49,7 @@ type tableItem struct {
 	tableID       int64
 	schemaVersion int64
 	tomb          bool
+	state         model.SchemaState
```

Comment:
This should be the value field instead of the key field?

Reply:
The non-public state is also a schema version ...

Reply:
Removed.